2-mandatory/4-space-colonies.js

```diff
@@ -15,7 +15,18 @@
   
 */
 
-function getSettlers() {}
+function getSettlers() {
+  function getSettlers(arr) {
+    let newArr =  [];
+  for(let el of arr){
+if (el.startsWith("A") && el.includes("family")){
```

Comment:
For family, you could have used .endsWith("family"). If not a string with "family" in the middle of the string would still pass this test.